packages/flutter/lib/src/painting/_image_provider_io.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I think this would be clearer if it was named `network_image_io.dart`. Similarly with `asset_image_io.dart`.